### DISCUSSION TITLE
Uninstall

### DISCUSSION THREAD

**Discussion by stav-ros** [nextpyp/.github#108] [Q&A] [answered]
Hey team! While trying to upgrade to the new version, a person from our team, followed the upgrade instructions and run the installation script from their Download directory, which created a new version of nextPYP. I deleted the files created in the local dir, but question is, are there any more files created elsewhere? I tried to re-install/upgrade the right nextPYP installation, but I cannot get it to work with the following error:

> Job for nextPYP.service failed because the control process exited with error code.
> See "systemctl status nextPYP.service" and "journalctl -xeu nextPYP.service" for details.

and the **_journalctl -xeu nextPYP.service_** output gives me:
```
The unit nextPYP.service completed and consumed the indicated resources.
apr 12 13:43:12 cryo1 systemd[1]: nextPYP.service: Start request repeated too quickly.
apr 12 13:43:12 cryo1 systemd[1]: nextPYP.service: Failed with result 'exit-code'.
░░ Subject: Unit failed
░░ Defined-By: systemd
░░ Support: http://www.ubuntu.com/support
░░ 
░░ The unit nextPYP.service has entered the 'failed' state with result 'exit-code'.
apr 12 13:43:12 cryo1 systemd[1]: Failed to start nextPYP.service - nextPYP.

```

Thanks for the help

**Comment by cuchaz** [comment, marked as answer]
Hi there. Unfortunately, the suggested log command doesn't tell us much useful information. It would be helpful to see more than a few lines of the log. Can you try this command `journalctl -n 200 -u nextPYP > service.log` and send the result?

If you're trying to abort an installation made in another location, it's possible the systemd service file was broken. You can find that at:
`/lib/systemd/system/nextPYP.service`

**Reply by stav-ros** [reply]
Thanks, fixing /lib/systemd/system/nextPYP.service was the key.
